### ISSUE TITLE
6.14.6 does not mark "+ +"  as wrong

### ISSUE THREAD

**Issue by toralf** [ansible/ansible-lint#3447]
I have in the ./inventory/group_vars:
```
or_port: "{{ range(1024, 65535) | random(seed=ansible_hostname + ipv4 + + ipv6 + seed_or_port) }}"
```
ans ansible-lint does not complain about it.

**Comment by ssbarnea** [comment]
I looked at what happens and I am afraid we cannot do much about it, mostly because in order to reach the error about about the bad operand, one needs to fist get all the variables defined and of the right type, something that is not possible during linting. In fact even Ansible core fails to spot the error when running syntax check and only fails at runtime.

Jinja is evaluated lazy, for good reasons. If someone finds a way to detect this, I would be more than happy to accept the patch, still until that I will close it as not planned.
